test/test_prior.py

```diff
@@ -68,8 +72,8 @@ def test_derivatives():
     Tests the finite gradients and second order derivatives.
     """
 
-    scales = ['lin', 'log', 'log10']
-    prior_types = ['uniform', 'normal', 'laplace', 'logNormal']
+    prior_types = ['normal', 'laplace', 'logNormal', 'parameterScaleUniform',
```

Comment:
Should `'uniform'` be added here? If not, then `if prior_type == 'uniform'` could be removed (a few lines below this).

Reply:
refactored all of the test, `uniform` was added to that list, but the optimization test doesn't make too much sense since a flat function doesn't have local minima :D

Reply:
Depends who you ask :see_no_evil:

Reply:
okay, well, it doesn't have a strict local minimum.